src/index.js

```diff
@@ -156,6 +156,64 @@ const stepTrends = $('#step-trends');
 const stepGoalChart = $('#step-goal-chart');
 const friendList = $('#friend-list');
 
+$(`.main__section--hydration`).on(`click`, () => {
+  event.preventDefault();
+  if (event.target.id === "activity-submit") {
+    fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData', {
+      method: 'POST',
+      headers: {
+              'Content-Type': 'application/json'
+        },
+      body: JSON.stringify({
+        userID: currentUserID,
+        date: $('#form__control--date1').val(),
+        numSteps: parseInt($('#form__control--steps').val()),
+        minutesActive: parseInt($('#form__control--minutes').val()),
+        flightsOfStairs: parseInt($('#form__control--stairs').val())
+      })
+    })
+    .then(response => response.json())
+    .then(data => console.log(data))
```

Comment:
This is super helpful to see that the response is happening, but we should probably take this out before submission.

Reply:
I agree, I was going to add on to this when I do some error handling.